templates/docker-compose.yml

```diff
@@ -81,7 +81,7 @@ services:
     logging: *default-logging
 
   postgres:
-    image: docker.io/postgres:16-alpine
+    image: pgautoupgrade/pgautoupgrade:16-alpine
```

Comment:
Would you mind pointing this to the official Docker Hub registry (ex, `docker.io/pgautoupgrade/pgautoupgrade`) (to avoid any issues with `registries.conf`, etc)?

Reply:
I updated.